### ISSUE TITLE
Send email to accepted teams announcing kickstarts

### ISSUE THREAD

**Issue by sedders123** [srobo/competition-team-minutes#147]
From https://github.com/srobo/competition-team-minutes/blob/master/SR2020/2019-09-20.md#specific

**Comment by sedders123** [comment]
Blocked on #144

**Comment by PeterJCLaw** [comment]
How does this differ from https://github.com/srobo/tasks/issues/314?

**Comment by sedders123** [comment]
It looks like it doesn't. Closing in favour of srobo/tasks#314
